wrftladj/solve_em_ad.F

```diff
@@ -672,7 +672,7 @@ SUBROUTINE solve_em_ad ( grid , config_flags  &
                              , ph_tendf, mu_tendf               &
                              , tke_tend                         &
                              , config_flags%use_adaptive_time_step &
-                             , curr_secs                        &
+                             , curr_secs, curr_mins2            &
```

Comment:
@tlspero Like curr_min2 in dyn_em/solve_em.F, it needs to be declared first. Somewhere near where curr_sec is declared is good. The same probably should be done for solve_em_tl.F too.